<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Battleship Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="youLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>You</string>
    </property>
   </widget>
   <widget class="QLabel" name="opponentLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Opponent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clientButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serverButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseModeLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressLineEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listenButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Listen!</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>351</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chatBrowser">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="chatLineEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>600</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>600</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shootButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>SHOOT!</string>
    </property>
   </widget>
   <zorder>listenButton</zorder>
   <zorder>addressLineEdit</zorder>
   <zorder>youLabel</zorder>
   <zorder>opponentLabel</zorder>
   <zorder>clientButton</zorder>
   <zorder>serverButton</zorder>
   <zorder>chooseModeLabel</zorder>
   <zorder>addressLabel</zorder>
   <zorder>portLabel</zorder>
   <zorder>portLineEdit</zorder>
   <zorder>connectButton</zorder>
   <zorder>textBrowser</zorder>
   <zorder>chatBrowser</zorder>
   <zorder>chatLineEdit</zorder>
   <zorder>sendButton</zorder>
   <zorder>shootButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
